frontend/src/app/firewood/forests/buy-firewood-permit/buy-firewood-permit.component.ts

```diff
@@ -0,0 +1,263 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute, Params, Router } from '@angular/router';
+import { FormBuilder, FormGroup, Validators, FormControl } from '@angular/forms';
+import { Title, Meta } from '@angular/platform-browser';
+import { Location } from '@angular/common';
+import * as moment from 'moment-timezone';
+import { NgxMdService } from 'ngx-md';
+import { alphanumericValidator } from '../../../application-forms/validators/alphanumeric-validation';
+import { currencyValidator } from '../../../application-forms/validators/currency-validation';
+import { emailConfirmationValidator } from '../../../application-forms/validators/email-confirmation-validation';
+import { lessThanOrEqualValidator } from '../../../application-forms/validators/less-than-or-equal-validation';
+import { FirewoodInfoService } from '../../_services/firewood-info.service';
+import { ApplicationFieldsService } from '../../../application-forms/_services/application-fields.service';
+import { FirewoodApplicationService } from '../../_services/firewood-application.service';
+import { UtilService } from '../../../_services/util.service';
+import { WindowRef } from '../../../_services/native-window.service';
+
+@Component({
+  selector: 'app-buy-firewood-permit',
+  templateUrl: './buy-firewood-permit.component.html'
+})
+export class BuyFirewoodPermitComponent implements OnInit {
+  forest: any;
+  permit: any;
+  submitted = false;
+  application: any;
+  applicationForm: FormGroup;
+  costPerTree: number;
+  apiErrors: any;
+  showRules = false;
+  jwtToken: string;
+  showCancelAlert = false;
+
+  constructor(
+    private route: ActivatedRoute,
+    private router: Router,
+    private location: Location,
+    private titleService: Title,
+    public formBuilder: FormBuilder,
+    // public markdownService: NgxMdService,
```

Comment:
I think we can probably get rid of this since we're not using markdown anymore.

Reply:
agreed removed